packages/compass-import-export/src/components/export-modal/export-modal.jsx

```diff
@@ -387,7 +390,7 @@ class ExportModal extends PureComponent {
           )}
         </Modal.Body>
         <Modal.Footer>
-          {this.renderBackButton()}
+          {this.props.isAggregation ? '' : this.renderBackButton()}
```

Comment:
Total nit, but this is a more common way to do this in React

```suggestion
          {!this.props.isAggregation && this.renderBackButton()}
```